.github/workflows/run_tests.yml

```diff
@@ -14,14 +40,15 @@ jobs:
       - name: "Clean & Assemble Debug"
         uses: eskatos/gradle-command-action@v1
         with:
-          wrapper-cache-enabled: true
+          distributions-cache-enabled: true
```

Comment:
`Warning: Input 'wrapper-cache-enabled' has been deprecated with message: Replaced by 'distributions-cache-enabled' which enables caching for all downloaded Gradle distributions`